static/js/compose.js

```diff
@@ -1134,7 +1134,7 @@ $(function () {
             break;
         case 'FileTooLarge':
             // sanitization not needed as the file name is not potentially parsed as HTML, etc.
-            msg = "\"" + file.name + "\"" + i18n.t(" was too large; the maximum file size is 25MiB.");
+            msg = "\"" + file.name + "\" " + i18n.t("was too large; the maximum file size is 25MiB.");
```

Comment:
Fair point! I like the idea that it gives context about the string, hadn't thought about that positive side effect. I'm changing it, then :)